package.json

```diff
@@ -4,7 +4,7 @@
 		"name": "Cartesia",
 		"url": "https://cartesia.ai"
 	},
-	"version": "1.3.0",
```

Comment:
Do we still need this to be alpha? Or can we release it as 1.4.0 with backward compat with the old clone method?

Reply:
the types are backwards compatible! I spent like an hour on that :P